### ISSUE TITLE
Link sharing

### ISSUE THREAD

**Issue by JulesGrd** [Martichou/rquickshare#34]
Automatically open a tab in the default browser if the shared content is a link

**Comment by Martichou** [comment]
Indeed, the current behavior (by design) is to automatically open the link.

I will keep this in my head and add an option to disable the automatic opening of links.

**Comment by JulesGrd** [comment]
OK, the main purpose was to reopen a previously shared link
